docs/app-areas.md

```diff
@@ -1,37 +1,28 @@
-An AEA is an intelligent agent whose goal is generating economic value for its owner. It can represent machines, humans (individuals or organisations), or data. 
+The <a href="../agent-oriented-development">agent-oriented development</a> guide presented some of the fundamental attributes of the environments which AEA-based solutions are suited for.
```

Comment:
```suggestion
The <a href="../agent-oriented-development">agent-oriented development</a> guide presents some of the fundamental attributes of the environments which AEA-based solutions are suited for.
```

Reply:
Past tense everywhere. No mixing of tenses.

Reply:
Done